src/azimuth/gui/screen.c

```diff
@@ -75,7 +80,7 @@ void az_init_gui(bool fullscreen, bool enable_audio) {
     "Azimuth",
     SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED,
     AZ_SCREEN_WIDTH, AZ_SCREEN_HEIGHT,
-    SDL_WINDOW_OPENGL | (fullscreen ? SDL_WINDOW_FULLSCREEN : 0));
+    SDL_WINDOW_RESIZABLE | SDL_WINDOW_OPENGL | (fullscreen ? SDL_WINDOW_FULLSCREEN : 0));
```

Comment:
With this PR I'm getting a strange behavior (on Mac OS X, at least; I haven't tested on other OSes yet) where if the game prefs are set to fullscreen on startup, then on startup the game image appears in the top-left 640x480 pixels of the screen, and the bottom and right halves of the screen are solid black, like this:
![Screen Shot 2021-03-06 at 11 17 50 AM](https://user-images.githubusercontent.com/35096/110213668-f5c89e00-7e6e-11eb-87a2-fdc2ad0ac829.png)
Switching to windowed mode and then back to fullscreen fixes it.  The problem also doesn't happen if the game starts in windowed mode and I then switch to fullscreen.

Not sure what's causing it; maybe it's somehow an artifact of the order in which we're changing the window settings?  While experimenting, I did discover that replacing `SDL_WINDOW_FULLSCREEN` on this line with `SDL_WINDOW_FULLSCREEN_DESKTOP` fixes the problem.  So that might be the easiest fix.

Reply:
Seems to be fixed now, thanks!